colossus/src/main/scala/colossus/core/Server.scala

```diff
@@ -390,9 +397,19 @@ object ServerStatus {
  *
  */
 object Server {
+
+  val MaxConnectionRegisterAttempts = 3
```

Comment:
I'm ok w/ this being hardcoded for now..I can't imagine a case right now where this needs to be configurable, since this is really just controlling a hopefully rare case of Worker dying.